app/models.py

```diff
@@ -424,6 +439,18 @@ def live_services(self):
     def domain_list(self):
         return [domain.domain for domain in self.domains]
 
+    def get_permissions_list(self):
+        from app.dao.organisation_permissions_dao import get_organisation_permissions_by_organisation_id
+
+        return [x.permission for x in get_organisation_permissions_by_organisation_id(self.id)]
+
+    def set_permissions_list(self, permissions):
+        from app.dao.organisation_permissions_dao import set_organisation_permission
+
+        permission_list = [OrganisationPermission(organisation_id=self.id, permission=p) for p in permissions]
+
+        set_organisation_permission(self, permission_list)
```

Comment:
This one I'm sort of OK with, but I think we could probably just bundle this all into `set_organisation_permission(organisation: Organisation, permissions: list[str])` and remove the extra helper layer

Reply:
Should this single function be in the dao?

Reply:
Yeah I think that's a good call